### PR TITLE
Enabling CI for UWP F5 runs

### PR DESCRIPTION
**Pull request by joperezr**
cc: @weshaggard @mmitche @danmosemsft @safern 

Now that our UWP F5 test run is clean, we are ready for adding this CI leg for uap tests. The only thing missing now is that the actual open target queue for ClientRS2 doesn't exist yet, but should be created very soon. Once that queue is ready, this PR will be ready to merge.

### CODE REVIEW

**Comment by joperezr** [comment]
@dotnet-bot test ci please

File: buildpipeline/windows.groovy [hunk 1]
```diff
@@ -80,15 +80,21 @@ simpleNode('Windows_NT','latest') {
...
+                {
+                    targetHelixQueues = ['Windows.10.Amd64.Open',
                                          'Windows.7.Amd64.Open',
                                          'Windows.81.Amd64.Open']
+                } else if (params.TGroup == 'uap') {
+                    targetHelixQueues = ['Windows.10.Amd64.ClientRS2.Open']
+                }
 
                 if (params.AGroup == 'x64') {
                     targetHelixQueues += ['Windows.10.Nano.Amd64.Open']
```
Comment (weshaggard) [inline buildpipeline/windows.groovy]: I don't think we want this for uap. probably best to add this under the netcoreapp section.

**Comment by weshaggard** [comment]
Not you don't need to test ci as changes to windows.groovy file directly take affect in the current ci runs. Only need to regenerate ci legs if you change pipelinejobs.groovy (or netci.groovy).

**Comment by joperezr** [comment]
Good to know, thanks @weshaggard. that also explains why the current CI is broken since the target queue is not ready.

File: buildpipeline/windows.groovy (hunk 1 above)
Comment (joperezr) [inline buildpipeline/windows.groovy]: good catch, I'll remove this

**Comment by joperezr** [comment]
@dotnet-bot test UWP CoreCLR x64 Debug Build please

**Comment by joperezr** [comment]
The problem with the UAP Tests now seems to be that the System.Text.RegularExpressions tests are hanging which is causing CI to fail. I'll try to investigate those before merging this.

**Comment by safern** [comment]
> The problem with the UAP Tests now seems to be that the System.Text.RegularExpressions tests are hanging which is causing CI to fail. I'll try to investigate those before merging this.

I've been investigating all day long and I'll already use the repro tool and ran them like ~15 times with it and like 5 without and haven't been lucky.

https://github.com/dotnet/corefx/issues/21547#issuecomment-311118850

Any ideas on how to hit it? It seems more like it is the test execution length than a hang. Maybe moving them to Outerloop for uap until we find the root cause to unblock the CI leg?

**Comment by danmosemsft** [comment]
@safern @joperezr I'm fine with moving this to outerloop for now if we can do that (a) for he whole library and (b) just for UAP somehow. I'm not sure there is.
Helix indicates it's timing out after 20 minutes, so if it's not approaching that long locally, I'm guessing there's something else going on. Perhaps @MattGal can help with machine acess.

**Comment by safern** [comment]
I talked offline with @joperezr and he suggested to disable them for uap. And we can do that in this PR and work with @MattGal to get a repro through getting access to the machine. 

Which one would you rather doy?

**Comment by danmosemsft** [comment]
Whatever gets this merge soonest...

**Comment by safern** [comment]
I'll disable them then and push it to this PR.

**Comment by safern** [comment]
Updated the pull request and disabled the tests for uap.

**Comment by danmosemsft** [comment]
LGTM
Note, ARM is failing still - sni.dll missing from the SQLClient package? But that won't be covered by ci anyway. I opened an issue.

**Comment by joperezr** [comment]
thanks for adding that @safern as soon as CI is green I'll hit merge

**Comment by joperezr** [comment]
Run is green, thanks everyone, change is in.

**Pull request merged**